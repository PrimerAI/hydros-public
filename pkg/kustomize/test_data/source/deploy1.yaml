apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: cert-manager
  name: cert-manager
  namespace: cert-manager
spec:
  replicas: 1
  selector:
    matchLabels:
      app: cert-manager
    metadata:
      labels:
        app: cert-manager
  template:
    spec:
      containers:
        - image: some/repo/gateway:latest
          name: gateway
          env:
            - name: SENTRY_KEY
              value: "1234a"
            - name: DD_AGENT_HOST
              valueFrom:
                fieldRef:
                  fieldPath: status.hostIP
            - name: SOMEENV
              value: helloworld
