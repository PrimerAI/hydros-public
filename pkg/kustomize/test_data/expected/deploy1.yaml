apiVersion: apps/v1
kind: Deployment
metadata:
  name: cert-manager
  namespace: cert-manager
  labels:
    app: cert-manager
spec:
  replicas: 1
  selector:
    metadata:
      labels:
        app: cert-manager
    matchLabels:
      app: cert-manager
  template:
    spec:
      containers:
        - name: gateway
          image: some/repo/gateway:latest
          env:
            - name: SOMEENV
              value: helloworld
